src/main/java/com/minecolonies/coremod/tileentities/TileEntityInfoPoster.java

```diff
@@ -0,0 +1,88 @@
+package com.minecolonies.coremod.tileentities;
+
+import net.minecraft.command.CommandResultStats;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.network.play.server.SPacketUpdateTileEntity;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.text.ITextComponent;
+import net.minecraft.util.text.TextComponentString;
+import net.minecraft.world.World;
+
+import javax.annotation.Nullable;
+
+public class TileEntityInfoPoster extends TileEntity
+{
+    public final ITextComponent[] signText =
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1170)